src/services/user.service.ts

```diff
@@ -0,0 +1,73 @@
+import {Users} from '../loaders/database';
+import {Op} from 'sequelize';
+import bcrypt from 'bcrypt';
+import {v4} from 'uuid';
+import {RESPONSE_MESSAGES} from '../config/messages';
+
+const SALT_ROUNDS = 10;
+const RETURN_ATTRIBUTES = ['id', 'login', 'age'];
+
+export default {
+  async getSuggestedUsers(loginSubstring: string, limit: number | undefined) {
+    return await Users.findAll({
+      where: {
+        login: {[Op.substring]: loginSubstring},
+      },
+      order: [['login', 'ASC']],
+      limit: limit,
+      attributes: RETURN_ATTRIBUTES,
+    });
+  },
+
+  async getUser(id: string) {
+    const user = await Users.findByPk(id, {
+      attributes: RETURN_ATTRIBUTES,
+    });
+    if (!user) {
```

Comment:
usually if you do a query operation (find something) you return found result or null\undefined and code, that calls the operation decide, if it needs to throw error.
I would return undefined and in controller added
```
if (!user) { res.status(404).send(RESPONSE_MESSAGES.USER_NOT_FOUND); }
```